.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Great job with adding `node_modules` to `.gitignore` 👏 Kindly add `test.md` also as mentioned in the requirements 👍